src/website/shared/management/commands/ingest_manual_evaluation.py

```diff
@@ -587,17 +587,17 @@ def ingest_maintainers(
         ms = []
         seen = set()
         for m in maintainers:
-            # Duplicate...
-            if m.name in seen:
-                continue
-
             # Maintainers without a GitHub or a GitHub ID cannot be reconciled.
             # This unfortunately creates a partial view of all maintainers of a
             # given package. If you want to fix this, you can start from
             # looking around https://github.com/NixOS/nixpkgs/pull/273220.
             if m.github is None or m.githubId is None:
                 continue
 
+            # Duplicate...
+            if m.githubId in seen:
```

Comment:
Depending on what gets merged first, we have to update one PR: https://github.com/Nix-Security-WG/nix-security-tracker/pull/103/files#diff-f0eaa811e0d9a58598296332294b179a7fb7967bc5f539c2afb01cc08ef3131fR596

Reply:
Better update the automatic change I guess